src/assets/wise5/services/studentAssetService.ts

```diff
@@ -13,6 +13,8 @@ export class StudentAssetService {
   audioFileExtensions = ['wav', 'mp3', 'ogg', 'm4a', 'm4p', 'raw', 'aiff', 'webm'];
   private showStudentAssetsSource: Subject<any> = new Subject<any>();
   public showStudentAssets$: Observable<any> = this.showStudentAssetsSource.asObservable();
+  private attachStudentAssetSource: Subject<any> = new Subject<any>();
+  public attachStudentAsset$: Observable<any> = this.attachStudentAssetSource.asObservable();
```

Comment:
Instead of a generic "any" type, what do you think about creating a new AttachStudentRequest interface?